Lombiq.Tests.UI/Extensions/BasicOrchardFeaturesTestingUITestContextExtensions.cs

```diff
@@ -132,44 +137,65 @@ public static UITestContext TestRegistrationWithInvalidData(this UITestContext c
 
             return context.ExecuteTest(
                 "Test registration with invalid data",
-                () =>
-                context.GoToRegistrationPage()
-                    .RegisterWith(model)
+                () => context
+                    .GoToRegistrationPage()
+                    .RegisterWith(parameters)
                     .ShouldStayOnRegistrationPage()
                     .ValidationMessages.Should.Not.BeEmpty());
         }
 
-        public static UITestContext TestRegistrationWithAlreadyRegisteredEmail(this UITestContext context, UserRegistrationModel model = null)
+        public static UITestContext TestRegistrationWithAlreadyRegisteredEmail(
+            this UITestContext context,
+            UserRegistrationParameters parameters = null)
         {
-            model ??= UserRegistrationModel.CreateDefault();
+            parameters ??= UserRegistrationParameters.CreateDefault();
 
             return context.ExecuteTest(
                 "Test registration with already registered email",
-                () =>
-                context.GoToRegistrationPage()
-                    .RegisterWith(model)
+                () => context
+                    .GoToRegistrationPage()
+                    .RegisterWith(parameters)
                     .ShouldStayOnRegistrationPage()
-                    .ValidationMessages[x => x.Email].Should.BeVisible());
+                    .ValidationMessages[page => page.Email].Should.BeVisible());
         }
 
-        public static UITestContext TestTurningFeatureOnAndOff(this UITestContext context, string featureName = "Background Tasks")
+        public static UITestContext TestContentOperations(this UITestContext context, string pageTitle = "Test page")
             =>
```

Comment:
There are a couple of other such examples in this class too.

```suggestion
        public static UITestContext TestContentOperations(this UITestContext context, string pageTitle = "Test page") =>
```

Reply:
Updated.